src/js/components/mapWidgets/userPointPopup.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { setRenderPopup } from 'js/store/appState/actions';
+
+import { RootState } from 'js/store';
+
+import 'css/popup.scss';
+
+const UserPointPopup = (): JSX.Element | null => {
+  const dispatch = useDispatch();
+  const [coordinates, setCoordinates] = useState<any | null>(null);
+
+  const userCoordinates = useSelector(
+    (state: RootState) => state.mapviewState.userCoordinates
+  );
+  const renderPopup = useSelector(
+    (state: RootState) => state.appState.renderPopup
+  );
+
+  useEffect(() => {
+    if (renderPopup && userCoordinates) {
+      const latitude = (userCoordinates as any).latitude
+        .toString()
+        .match(/^-?\d+(?:\.\d{0,2})?/)[0];
```

Comment:
what does this regex do?

Reply:
This returns the last 2 decimals without rounding up or down. For example, `14.123456` becomes `14.12`

Reply:
I would rather increase precision to 5 decimals than use regex that we have to document somewhere. `const latitude = userCoordinates.latitude.toFixed(5)` ?

Reply:
Excellent point - consider it done!